main.go

```diff
@@ -140,6 +142,17 @@ var (
 	membersCache []User
 )
 
+// blacklistedMemebers returns a list of login names of members which should
+// not be included in pull requests.
+func blacklistedMembers() map[string]bool {
+	ret := map[string]bool{}
```

Comment:
It was not to explain how it works :/, its to show its an idiom suggested by the official blog